src/scribe_data/wikidata/SPARQL_QUERY_WRITING.md

```diff
@@ -0,0 +1,99 @@
+# SPARQL Query Writing for Wikidata Lexemes
+
```

Comment:
Hi @OkpePhillips Thanks for the documentation. Looks accurate and awesome. though I agree with @andrewtavis that it would be nice to include the link to the WIKIDATAGUIDE in the file. It will give the details of the terminologies and the required stuff needed to know to proceed with query writing. What do you think?

```suggestion
Wikidata is a free and open knowledge base that provides structured data to support a wide range of applications, including linguistic data through lexemes. SPARQL queries enable powerful searches and extraction of specific data from this repository, such as lexeme forms and their grammatical features.

To learn more, visit the [Wikidata Guide](https://github.com/scribe-org/Organization/blob/main/WIKIDATAGUIDE.md).
```